.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# these owners will be the default owners for approving PRs to merge into the `master` branch
+
+*       @master-approvers
```

Comment:
this seems to be wrong according to GH

Reply:
I merged with master before change the branch base. I'll revert this part, thanks!

Reply:
@Twsouza, I have just checked this file and it's the same as master. Can I ignore this warning?